ref/anbox-https-api.md

```diff
@@ -491,3 +494,211 @@ Platform | Field name       | Available since   | JSON type | Access | Descripti
 `webrtc` | `rtc_log`         | 1.15 | Boolean   | read/write | Enable/disable [RTC event logging](https://webrtc.googlesource.com/src/+/lkgr/logging/g3doc/rtc_event_log.md). Logs are written to `/var/lib/anbox/traces/rtc_log.*` inside the instance. |
 `webrtc` | `stream_active`   | 1.15 | Boolean   | read | `true` if a client is actively streaming, `false` if no client is connected. |
+
+
+<a name="heading--10vhal"></a>
+### `/1.0/vhal`
+
+This endpoint queries the [Android VHAL](https://source.android.com/docs/automotive/vhal)
+through Anbox. It mimics the
+[VHAL HIDL interface](https://source.android.com/docs/automotive/vhal/hidl-vhal-interface)
+for `get` and `set` and follows RESTful API conventions. All queries on this
+endpoint will fail with a 500 error code on non-automotive Anbox images.
+
+#### GET `1.0/vhal/{prop_id}/{area_id}`
+ * Description: Get a VHAL property value
+ * Operation: sync
+ * Return: Current value for requested property
+ * Parameters:
+    * `prop_id`: Property identifier. Can be given in decimal, octal or hexadecimal format.
+    * `area_id`: Valid area identifier for the property. Can be omitted for global properties. Can be given in decimal, octal, or hexadecimal format.
+
+To get the list of available properties and areas, query first the [`1.0/vhal/config` endpoint](#heading--10vhalconfig).
+
+Example return value for `curl -s -X GET --unix-socket /run/user/1000/anbox/sockets/api.unix s/1.0/vhal/0x15600503/0x31 | jq .`:
+
+```bash
+{
+  "metadata": {
+    "value": {
+      "area_id": 49,         # Requested area.
+      "bytes": [],           # Raw bytes value as an 8-bit unsigned integer array.
+      "float_values": [      # Float array
+          16.0
+      ],
+      "int32_values": [],    # 32-bits signed integer array
+      "int64_values": [],    # 64-bits signed integer array
+      "prop": 358614275,     # Requested property.
+      "status": 0,           # Status of the property, see below.
+      "string_value": "",    # UTF-8 string value.
+      "timestamp": 0         # Time when the property was last set in nanoseconds since boot.
+    }
+  },
+  "status": "Success",
+  "status_code": 200,
+  "type": "sync"
+}
+```
+
+Usually, only one of `bytes`, `float_values`, `int32_values`, `int64_values`,
+`string_value` is set, and the rest is empty or omitted, depending on the
+property type (see [`1.0/vhal/config`](#heading--10vhalconfig)).
+`MIXED` property types may have multiple of these values set at the same time, see
+[VHAL property types](https://source.android.com/docs/automotive/vhal/property-configuration#property-types).
+
+Status can be one of the following values, taken from the
+[VhalPropertyStatus](https://cs.android.com/android/platform/superproject/+/android10-release:hardware/interfaces/automotive/vehicle/2.0/types.hal;l=2700-2720)
+enumeration:
+
+|Name|Value|Description|
+|-|-|-|
+|Available|0|Property is available and behaving normally|
```

Comment:
Very small nitpick. Better to have punctuation at the end of all sentences or not have have them for all.